base/compiler/abstractinterpretation.jl

```diff
@@ -3113,6 +3113,7 @@ end
 abstract_eval_ssavalue(s::SSAValue, sv::InferenceState) = abstract_eval_ssavalue(s, sv.ssavaluetypes)
 
 function abstract_eval_ssavalue(s::SSAValue, ssavaluetypes::Vector{Any})
+    (s.id < 1 || s.id > length(ssavaluetypes)) && throw(InvalidIRError())
```

Comment:
```suggestion
    (1 ≤ s.id < length(ssavaluetypes)) || throw(InvalidIRError())
```
would be a bit neater?

Reply:
I actually didn't know we had that syntax! Thanks for the tip